CHANGELOG.md

```diff
@@ -30,17 +30,18 @@ All changes to the project will be documented in this file.
     /u:System.IO
     ```
     and the following `omnisharp.json`:
-     
+
     ```
     {
         "Script": {
             "RspFilePath": "path/to/my.rsp"
         }
     }
-    ```   
+    ```
```

Comment:
These are changes you want, [according to the .editorconfig](https://github.com/OmniSharp/omnisharp-roslyn/blob/e8507177962cda27804f930dc58bce090f910c75/.editorconfig#L6). 😇

Side note though; I often enable `trim_trailing_whitespace = true` for certain files only because Markdown relies on a double trailing space to indicate a line break.